scripts/stablex/get_auction_elements.js

```diff
@@ -1,14 +1,110 @@
 const BatchExchange = artifacts.require("BatchExchange")
+const BN = require("bn.js")
+const argv = require("yargs")
+  .option("valid", {
+    type: "boolean",
+    describe: "Only show valid and future orders",
```

Comment:
Should this be the default behaviour? Maybe add a `--all` flag to include invalid orders?